source/css/_layout.wrappers.scss

```diff
@@ -46,6 +46,28 @@
   max-width: rem(1250);
 }
 
+.l-container-grid--7-col {
+  position: relative;
+
+  &::before {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    margin-right: auto;
+    margin-left: auto;
+    width: calc(100% - (2 * var(--offset)));
```

Comment:
I **LOVE** that you're using native CSS variables, but we'll have to provide a fallback since they're [not yet supported in any IE/Edge browser](https://caniuse.com/#feat=css-variables) just yet.